cmd/oslo/convert/convert.go

```diff
@@ -50,44 +49,34 @@ You can also convert a directory of files:
 
   oslo convert -d path/to/directory -o nobl9
 
-The output is written to standard output.  If you want to write to a file, you can redirect the output:
+The output is written to standard output. If you want to write to a file, you can redirect the output:
```

Comment:
Interesting idea, but I think that for a string that is basically documentation this additional jump to another file when your changing something in this may be error-prone. I mean both implementation and reviewer may forget about updating help and when it's here they see it explicitly

Reply:
Fine